tensorbay/opendataset/UrbanObjectDetection/loader.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name, missing-module-docstring
+
+"""Dataloader of UrbanObjectDetection dataset."""
+
+import os
+
+from tensorbay.dataset import Dataset
+from tensorbay.opendataset._utility import get_voc_detection_data
+
+_SEGMENT_NAMES = ("train", "val", "test")
+_BOOLEAN_ATTRIBUTES = {"difficult", "truncated"}
```

Comment:
can we get `_BOOLEAN_ATTRIBUTES` from catalog, rather than a constant?

Reply:
We decide not to get _BOOLEAN_ATTRIBUTES from catalog.

Reply:
Why?

Reply:
ask Yexuan